### DISCUSSION TITLE
Is there a place I can follow changes to the GraphQL API?

### DISCUSSION THREAD

**Discussion by kartikdc** [community/community#24644] [Apps, API and Webhooks]
<p>Hello.</p>
<p>This weekend, I think something was changed in the GraphQL API. IDs used to be of type String, but it was changed to a different type called ID. Because of this, some of my automation tools failed and it took me some time to debug the issue.</p>
<p>Is there a place where I can follow similar changes made to the API? It would save me some trouble.</p>
<p>Thanks. :slight_smile:</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>you mean, this one ?</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/graphql/overview/changelog">
  <header class="source">
      <img src="https://docs.github.com/assets/cb-803/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/graphql/overview/changelog" target="_blank" rel="noopener">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/47092464/181085814-c68f075a-6e07-4373-bffb-3180669a6da0.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/graphql/overview/changelog" target="_blank" rel="noopener">Changelog - GitHub Docs</a></h3>

  <p>The GraphQL schema changelog is a list of recent and upcoming changes to our GraphQL API schema. It includes backwards-compatible changes, schema previews, and upcoming breaking changes.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by kartikdc** [comment]
<p>Oh wow. No idea how I missed that. Thanks.</p>
<p>Looking at the change log it doesn’t say anything about the change I mentioned but it references type ID a while back so I think it’s just some deprecation kicking in.</p>
